packages/popover/src/Arrow.js

```diff
@@ -12,7 +12,7 @@ const PopoverArrow = ({
 }) => (
   <Arrow
     className={className}
-    innerRef={arrowProps.ref}
+    ref={arrowProps.ref}
```

Comment:
Hey @qualitymanifest, could you leave the old `innerRef` in addition to `ref`? Someone can correct me if I'm wrong, but we still need the old once since we support `styled-components@^3` as a peer, so we have to keep backward compatibility.

Reply:
Sure, I'll add these back in a second

Reply:
actually, i don't think that is necessary. react 16.3 is already our floor and it supports forwardRefs